website/server/models/task.js

```diff
@@ -174,6 +174,21 @@ TaskSchema.methods.scoreChallengeTask = async function scoreChallengeTask (delta
   await chalTask.save();
 };
 
+// skips challenge and group tasks
+export let skipChallengeTasks = [
```

Comment:
Can you add the word qurey params to this? Maybe `skipChallengeTasksQuery`? And, I think this is technically a constant.